### DISCUSSION TITLE
run script across existing files

### DISCUSSION THREAD

**Discussion by wjbeckett** [RandomNinjaAtk/docker-lidarr-extended#122] [Q&A] [answered]
Is there a way to run the processing script across existing files to tag/grab lyrics?

**Comment by RandomNinjaAtk** [comment]
Lyrics come from the initial downloads from the download client, so there isn't really a way to do that unless its something you can accomplish with beets.

**Comment by wjbeckett** [comment, marked as answer]
Okay, I understand. So to have everything nicely tagged, etc, I'd need to remove file and re-grab with the script.

**Reply by RandomNinjaAtk** [reply]
Yeah, likely the simplest way.

**Reply by RandomNinjaAtk** [reply]
@Backslashh 
Just thought you should know this, one thing to keep in mind, lyrics only work for lossless (flac) and opus output formats,  aac/mp3 are not supported.
